examples/magento-graphcms/pages/search/[[...url]].tsx

```diff
@@ -57,6 +57,11 @@ function SearchResultPage(props: Props) {
             : i18n._(/* i18n */ 'Search')
         }
         metaRobots={['noindex']}
+        metaDescription={
+          search
+            ? i18n._(/* i18n */ 'Results for ‘{search}’', { search })
+            : i18n._(/* i18n */ 'Search')
+        }
```

Comment:
Page is not indexable, metaDescription is useless. Adding this change creates useless churn